controllers/user.go

```diff
@@ -87,6 +87,41 @@ func (c *ApiController) GetUser() {
 	id := c.Input().Get("id")
 	owner := c.Input().Get("owner")
 	email := c.Input().Get("email")
+	organization := object.GetOrganization(fmt.Sprintf("%s/%s", "admin", strings.Split(id, "/")[0]))
+	if organization.UseAuthForUserApi {
+		requestUserId := c.GetSessionUsername()
+		if requestUserId == "" {
+			c.ResponseError("Please login first.")
+			return
+		}
+		targetUser := object.GetUser(id)
+		if targetUser == nil {
+			c.ResponseError(fmt.Sprintf("The user: %s doesn't exist", id))
+			return
+		}
+
+		hasPermission := false
+		if strings.HasPrefix(requestUserId, "app/") {
+			hasPermission = true
+		} else {
+			requestUser := object.GetUser(requestUserId)
+			if requestUser == nil {
+				c.ResponseError("Session outdated. Please login again.")
+				return
+			}
+			if requestUser.IsGlobalAdmin {
```

Comment:
We better use Casbin authorization to control permission here

Reply:
but we want to have this permission changable with every organisation, using casbin auth will require declaring at a global level, with no choice,
i have used similar approach as used in set-password api to verify permissions